lib/dRep/dRepImpl.py

```diff
@@ -38,40 +56,50 @@ class dRep:
     # the latter method is running.
     ######################################### noqa
     VERSION = "0.0.1"
-    GIT_URL = ""
-    GIT_COMMIT_HASH = ""
+    GIT_URL = "https://github.com/n1mus/dRep.git"
+    GIT_COMMIT_HASH = "05f421114e50a04008d9897cdb20399b23a47b46"
 
     #BEGIN_CLASS_HEADER
     #END_CLASS_HEADER
 
     # config contains contents of config file in a hash or None if it couldn't
     # be found
     def __init__(self, config):
-        #BEGIN_CONSTRUCTOR
+####################################################################################################
+#---------------------------------------------------------------------------------------------------
+#---------------------------------------------------------------------------------------------------
+#---------------------------------------------------------------------------------------------------
+       #BEGIN_CONSTRUCTOR
         self.callback_url = os.environ['SDK_CALLBACK_URL']
         self.workspace_url = config['workspace-url']
         self.srv_wiz_url = config['srv-wiz-url']
         self.shared_folder = config['scratch']
+        self.testData_dir = '/kb/module/test/data'
```

Comment:
Is there a directory you can pull out of config to use, rather than hardcoding '/kb/module' here?

Reply:
i don't think so